sematic/ui/packages/common/src/component/Table.tsx

```diff
@@ -47,33 +47,51 @@ const StyledHeader = styled(TableHead, {
         margin-left: -${theme.spacing(5)};
         background: ${theme.palette.p3border.main};
     }
+
+    & th:first-of-type {
+        padding-left: ${theme.spacing(5)};
+    }
 `;
 
 const TableDataRow = styled(TableRow)`
     height: 50px;
+    cursor: pointer;
+
+    &:hover {
+        td {
+            background: ${theme.palette.p3border.main};
+        }
+    }
 
     & td {
         padding-left: 0;
-        padding-right: ${theme.spacing(2.5)};
+        padding-right: ${theme.spacing(5)};
         padding-top: 0;
         padding-bottom: 0;
         border-bottom: none;
 
         &:last-of-type {
             padding-right: 0
         };
+
+        &:first-of-type {
+            padding-left: ${theme.spacing(5)};
```

Comment:
Should 5 be extracted to a constant?

Reply:
Thanks. Might not be very practical. There are such numbers everywhere, some are related, and some are not despite using the same value. 

In reality, any UI defect might have to be investigated separately, and those numbers might have to be tuned individually. Using constants might be too much of a burden.